scripts/moteusConfigConvert.py

```diff
@@ -0,0 +1,80 @@
+"""
+Script for converting moteus config into a writable format.
+Takes in file generated from:
+
+sudo moteus_tool -t X --dump-config
+
+and adds "conf set " to every line. Generates "filename_write.cfg"
+Write the config to a moteus with:
+
+sudo moteus_tool -t X --write-config filename_write.cfg
+
+Program input prompts for moteus ID, and aux pin configurations.
+
+"""
+
+
+def main():
+    print("To save a particular configuration, use sudo moteus_tool -t X --dump-config > filename.cfg")
+    print("Enter source config filename: ")
+    input_filename = input()
+    output_filename = input_filename.replace(".cfg", "") + "_write.cfg"
+
+    with open(input_filename, "r") as input_file:
+        output_file = open(output_filename, "w+")
+
+        print("Enter target moteus ID:")
+        target_id = int(input())
+        print("Default configuration is aux2 pin0 as hall pin.")
+        print("Would you like to configure aux2 pin1 as a hall pin instead? (y/n)")
+        aux2_config = input()
+
+        pin0mode = 6
+        pin0pull = 1
+        pin1mode = 0
+        pin1pull = 0
+        if aux2_config == "y":
+            pin0mode = 0
+            pin0pull = 0
+            pin1mode = 6
+            pin1pull = 1
+
+        if target_id >= 0 or target_id <= 8:
```

Comment:
Might be nice to check target_id immediately and return if it's bad. Then you don't have to have this extra layer of indentation

Reply:
Done